website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,103 @@
+are also utilized when there are multiple production-level auth methods that need to be kept in separate scopes.
+- **Project** - Project scopes are contained in Org scopes and contain infrastructure resources.
+
+#### Auth Methods
+[Auth Methods](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/auth-methods) are resources that provide a mechanism for users to authenticate to 
+Boundary. Auth methods can only be contained by the global or Org scope. The available auth methods are:
+- Username/Password
+- [OIDC](https://developer.hashicorp.com/boundary/tutorials/access-management/oidc-auth)
+
+#### Accounts
+[Accounts](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/accounts) are resources that represent a set of credentials used to establish the 
+identity of a user. An account is created with an auth method, and can be associated with a single user. Accounts can be contained by any scope.
+
+#### Users
+[Users](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/users) are resources that represent an individual person or entity for the purposes of 
+access control. A user can contain multiple accounts from multiple auth methods, and can be contained by any scope.
+
+#### Groups
+[Groups](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/groups) are resources that represent a collection of users, and can be contained by any 
+scope. [Managed groups](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/managed-groups) are a special kind of group that syncs with the auth 
+method's identity provider to provide up to date information.
+
+#### Roles
+[Roles](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/roles) are resources that contain a collection of 
+[permissions](https://developer.hashicorp.com/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles 
+can be contained by any scope, and can specify which scope the permissions will apply to.
+
+## Access Management
+#### Configuring Users
+When adding new users to Boundary using the password auth method, there are 3 steps to follow. Note: for OIDC auth methods, the following will be auto-generated using 
+information provided by the identity provider.
+1. **Auth Method** - Choose the scope and the auth method that the user will use to authenticate to Boundary. If the desired auth method does not exist, a new one needs 
+to be created.
+2. **Account** - After the auth method is selected, an account can be created under the auth method.
+3. **User** - In the same scope, create a new user, and then attach the new account to the new user. The user is now able to authenticate to Boundary.
+
+#### Granting Permissions
+When setting up access controls for a user, it is important to first consider which scopes the user needs access to. **Each scope needs its own role** which the user must 
```

Comment:
Hum... I'm not sure I'd approach the problem in this manner.  

It really depends what users/accounts/resources I'm dealing with.  

For OIDC accounts, I'd probably create managed groups and then create
appropriate roles and finally associate managed groups with roles.

Even if I were just dealing with static accounts/users/groups I probably would
still create groups and not assign a user directly to a role.  

Perhaps expand this section with more use-cases?

Reply:
Good call, will address this